### DISCUSSION TITLE
Tooltips not allowing focus to text fields due to preventDefault

### DISCUSSION THREAD

**Discussion by bdow** [adobe/react-spectrum#3058] [General]
Hi all,

We have text fields in a table where showing an error message would be best if we can do it in a tooltip. However, there is code in the `usePress` file that will `preventDefault` [onPointerDown events](https://github.com/adobe/react-spectrum/blob/cc158b410d11894fb2f14cc2732165dd91786036/packages/%40react-aria/interactions/src/usePress.ts#L324). This causes text fields wrapped inside of a tooltip trigger to not be accessible via a left click. 
Here is a [codesandbox](https://codesandbox.io/s/clever-brown-51vgp1?file=/src/App.js) to demonstrate the issue. The only way I've figured out to get around this is by pretending the trigger element is draggable, as that is the only workaround to not call `e.preventDefault`. 
Please let me know if I am going about this wrong... Thanks!

**Comment by snowystinger** [comment]
Hey, thanks for the discussion. We actually do not support tooltips on text fields, it's very difficult for screen readers to access and can be quite distracting for a mouse or touch user as well. See this for a bit more info on potential other conflicts https://github.com/adobe/react-spectrum/issues/1687#issuecomment-797838249

I suggest following what our Design team recommends, using a ContextualHelp or HelpText type of approach https://spectrum.adobe.com/page/text-field/#Help-text-(description-and-error-message) and https://spectrum.adobe.com/page/contextual-help/

**Reply by bdow** [reply]
Thanks for the quick reply @snowystinger. Based on your feedback, I think it may be a good idea to warn/error if the user is attempting to wrap a textfield element within a tooltip. Or maybe add this limitation to the JSDoc for `useTooltipTrigger`. Thanks again for the explanation.
